src/Apps/Sell/Routes/ConsignmentInquiry/ConsignmentInquiryConfirmation.tsx

```diff
@@ -13,7 +13,7 @@ export const ConsignmentInquiryConfirmation: React.FC = () => {
       <Spacer y={4} />
 
       <Button
-        width={["100%", "20%"]}
+        width={["100%", "auto"]}
```

Comment:
Threw this in for free:

| Before | After |
|--------|--------|
| ![Screenshot 2024-11-19 at 4 14 38 PM](https://github.com/user-attachments/assets/963cd09f-f749-4756-88d1-a1977b73dbf0) | ![Screenshot 2024-11-19 at 4 15 51 PM](https://github.com/user-attachments/assets/69b0fa31-175d-4c02-9186-ba1fa46113ba) |